client/src/services/stardust/stardustTangleCacheService.ts

```diff
@@ -672,6 +674,17 @@ export class StardustTangleCacheService extends TangleCacheService {
             { error: response.error };
     }
 
+    /**
+     * Get the participation events details.
+     * @param request The request to send.
+     * @returns The response from the request.
+     */
+    public async participationEventDetails(request: IParticipationEventRequest): Promise<IParticipationEventResponse> {
```

Comment:
If we are not going to "cache it", consider not even adding this to `tangleCacheService` and just calling directly from `apiClient` where this is used.